flask/tests/test_validation.py

```diff
@@ -0,0 +1,32 @@
+""" test validators """
+from unittest.mock import patch
+from unittest import TestCase
+from werkzeug.exceptions import UnsupportedMediaType
+from app.decorators import validate_json
+
+
+class ValidationTest(TestCase):
```

Comment:
I like where this class is going! As an aside, I'm not sure if it's possible or really that good of a practice to mock out the ORM or MinIO over full integration tests though. I have some concerns about how long the test suites take now and they're not really do much yet.